gpu/test_install_gpu_driver.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+from parameterized import parameterized
+
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class NvidiaGpuDriverTestCase(DataprocTestCase):
+  COMPONENT = 'gpu'
+  INIT_ACTION = 'gs://dataproc-initialization-actions/gpu/install_gpu_driver.sh'
+  MASTER_GPU_TYPE = 'type=nvidia-tesla-v100'
+  WORKER_GPU_TYPE = 'type=nvidia-tesla-v100'
+
+  @classmethod
+  def setUpClass(cls):
+    super().setUpClass()
+    _, region, _ = cls.run_command("gcloud config get-value compute/region")
+    cls.REGION = region.strip() or "global"
+
+  def verify_instance(self, name):
+    ret_code, stdout, stderr = self.run_command(
+      'gcloud compute ssh {} --command '
+      '"nvidia-smi"'.format(name)
+    )
+    self.assertEqual(ret_code, 0,
+                     "Failed to validate cluster. Error: {}".format(stderr))
+
+  def verify_instance_gpu_agent(self, name):
+    ret_code, stdout, stderr = self.run_command(
+      'gcloud compute ssh {} --command '
+      '"systemctl status gpu_utilization_agent.service"'.format(name)
+    )
+    self.assertEqual(ret_code, 0,
+                     "Failed to validate cluster. Error: {}".format(stderr))
+
+  @parameterized.expand([
+    ("STANDARD", "1.3", ["m"], MASTER_GPU_TYPE, WORKER_GPU_TYPE),
+  ], testcase_func_name=DataprocTestCase.generate_verbose_test_name)
```

Comment:
Do we only support 1.3? If so, document it.

Reply:
Need to check which versions we have tested this, I only test it with 1.3-deb9. Will add a note.

Reply:
Found this is supported in Dataproc 1.0, 1.1, 1.2 and 1.3 Debian 9.